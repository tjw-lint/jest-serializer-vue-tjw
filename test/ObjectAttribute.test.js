import { shallowMount } from '@vue/test-utils';
import ObjectAttribute from './components/ObjectAttribute.vue';

describe('ObjectAttribute.vue', () => {
  test('Snapshot unchanged', () => {
    const wrapper = shallowMount(ObjectAttribute);

    expect(wrapper)
      .toMatchSnapshot();
  });
});
