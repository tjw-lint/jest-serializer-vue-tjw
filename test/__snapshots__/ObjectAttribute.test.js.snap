// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ObjectAttribute.vue Snapshot unchanged 1`] = `
<div title="{a:true,b:'Some \\'text\\'',c:{d:{e:'23'},f:'\\'&quot;\\'&quot;'}}">
  <h1 title="['asdf','qwer']">
    text
  </h1>
</div>
`;
